src/map/packets/action.h

```diff
@@ -58,7 +58,8 @@ enum ACTIONTYPE : uint8
     ACTION_RANGED_INTERRUPT     = 32,
     ACTION_MOBABILITY_START     = 33,
     ACTION_MOBABILITY_INTERRUPT = 35,
-    ACTION_RAISE_MENU_SELECTION = 37
+    ACTION_RAISE_MENU_SELECTION = 37,
+    ACTION_JOBABILITY_INTERRUPT = 38,
```

Comment:
Not really YOUR problem, or this PR problem.

However, what are this numbers?
They seem random and with jumps between them, and yet, theres a comment in there saying they aren't IDs?

Reply:
lua land has a bunch more, they're not defined in C++ for some reason. Was planning to backfill that before undrafting

Reply:
Oh and for the "not IDs thing", they don't map to real action categories but eventually do get mapped to them (such as shown in this PR in action.cpp)

Reply:
Taking back that statement about "backfilling before undrafting" -- it looks like there's more here than I knew at a glance. there's some overlap in lua that doesn't match c++...